ocs_ci/templates/workloads/cephfs_stress/cephfs_filesystem_monitor.py

```
@@ -0,0 +1,633 @@
+    for name, cmd in commands.items():
+        try:
+            result = subprocess.run(cmd, shell=True, capture_output=True, text=True, timeout=10)
+            output = result.stdout if result.returncode == 0 else result.stderr
+            
+            # OPTIMIZED: Truncate large outputs to save memory
+            if len(output) > MAX_OUTPUT_SIZE:
+                output = output[:MAX_OUTPUT_SIZE] + f"\n... [TRUNCATED {len(output) - MAX_OUTPUT_SIZE} bytes]"
+            
+            state[name] = output
+        except subprocess.TimeoutExpired:
+            state[name] = f"Command timed out after 10s"
```

Ruff (0.15.12)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)